packages/storybook/src/Select/Select.stories.tsx

```diff
@@ -52,3 +61,18 @@ export function Initial() {
     </Select>
   )
 }
+
+export function Loading() {
+  const { getToggleButtonProps } = useSelect<string>({ items })
+
+  return (
+    <SelectButton isOpen={false} {...getToggleButtonProps()}>
+      <Flex sx={{ width: '100%' }}>
+        <Box sx={{ flex: 1 }}>
+          <Text sx={{ color: 'gray.3' }}>Loading</Text>
+        </Box>
+        <DotLoadingIndicator sx={{ mr: 1, mt: 1 }} />
```

Comment:
Looks like there's one more `mt` that we could remove:

```suggestion
        <DotLoadingIndicator sx={{ mr: 1, mt: 1 }} />
```